src/Tools/dotnet-monitor/Exceptions/EventExceptionsPipelineNameCache.cs

```diff
@@ -63,8 +63,70 @@ public bool TryGetExceptionId(ulong exceptionId, out ulong exceptionClassId, out
             return true;
         }
 
+        public bool TryGetClassId(ulong classId, out ClassData data)
```

Comment:
All of these new methods don't really provide much value because you can already do the same thing from the already publicly available `NameCache` property.